### PR TITLE
Update tests for PHPUnit 5.5

### PR DESCRIPTION
**Pull request by MartinPeverelli**

### CODE REVIEW

**Comment by MartinPeverelli** [comment]
@Nyholm I updated the PR, but it seems to me that the TravisCI error is due to the minimum stability setting and the dependencies. :(

File: test/Github/Tests/ResultPagerTest.php [hunk 1]
```diff
@@ -97,9 +101,12 @@ public function testFetch()
         $result = 'foo';
         $method = 'bar';
         $parameters = array('baz');
-        $api = $this->getMock('Github\Api\ApiInterface');
+        $api = $this->createMock('Github\Api\ApiInterface');
```
Comment (kokspflanze) [inline test/Github/Tests/ResultPagerTest.php]: `createMock` was added in PHPUnit 5.4 and PHPUnit 5.X is min PHP 5.5

https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-5.4.0#new-features

**Comment by GrahamCampbell** [comment]
👎

File: test/Github/Tests/ResultPagerTest.php (hunk 1 above)
Comment (xabbuh) [inline test/Github/Tests/ResultPagerTest.php]: You can use `getMockBuilder()` instead which works on all supported PHPUnit versions without triggering deprecations.

**Comment by Nyholm** [comment]
@GrahamCampbell Could you elaborate your comment?

**Comment by GrahamCampbell** [comment]
My comment was basically because of:

> createMock was added in PHPUnit 5.4 and PHPUnit 5.X is min PHP 5.6

However, if the solution:

> You can use getMockBuilder() instead

is implemented instead, then 👍.

**Pull request merged**